packages/babel-generator/src/buffer.js

```diff
@@ -85,6 +84,15 @@ export default class Buffer {
     this._queue.unshift([str, line, column, identifierName, filename]);
   }
 
+  _trimEnd(str: string): string {
```

Comment:
Could we throw a comment on here so it is clear why we do this rather than Lodash? Over time it'll be easy to lose info about which issue introduced this, and why.